sechub-wrapper-prepare/src/main/java/com/mercedesbenz/sechub/wrapper/prepare/modules/skopeo/SkopeoWrapper.java

```diff
@@ -114,10 +114,13 @@ private ProcessBuilder buildProcessDownload(SkopeoContext context) {
         List<String> commands = new ArrayList<>();
 
         String location = locationConverter.convertLocationForDownload(context.getLocation());
+        String additionalTag = locationConverter.removeProtocolPrefix(context.getLocation());
         File downloadDirectory = context.getToolDownloadDirectory().toFile();
 
         commands.add("skopeo");
         commands.add("copy");
+        commands.add("--additional-tag");
+        commands.add(additionalTag);
```

Comment:
see my comments at location converter - we must ensure, that additional tag can never be null/empty or we handle the situation and do a conditional add here